python/mxnet/gluon/nn/conv_layers.py

```diff
@@ -86,10 +121,20 @@ class _Conv(HybridBlock):
         Initializer for the `weight` weights matrix.
     bias_initializer: str or `Initializer`
         Initializer for the bias vector.
+    cudnn : str,
+        Supported values - {'off', 'default', 'fastest', 'limited_workspace', 'global'}.
```

Comment:
by default it should align with env variable, i.e., the global in your revision. And I don't quite like the name global, sounds like a global tuning option. Is env a better name?

Reply:
Thanks.
default -> is actual default now i.e., uses environment variable.
no_autotune -> Uses default cudnn algorithm.

So basically,
global is renamed to default,
default is renamed to no_autotune.

@zhreshold - Let me know if changes looks good. Thanks.